libraries/chain/include/eosio/chain/config.hpp

```diff
@@ -22,6 +22,7 @@ const static auto forkdb_filename            = "forkdb.dat";
 const static auto default_state_size            = 1*1024*1024*1024ll;
 const static auto default_state_guard_size      =    128*1024*1024ll;
 
+const static uint64_t primary_index_name       = N(primary);
```

Comment:
Don't use eosio include files for chaindb defines.
You can use `cyberway/chaindb/common.hpp` or `cyberway/chaindb/names.hpp` for such cases.